frontend/src/main/input.tdot

```diff
@@ -1,54 +1,36 @@
-create monthly_budget {
-  date: September 2019,
-  expenses: {
-	"eating out": { budget: 200, track: true },
-	coffee: { budget: 80, track: true },
-	transport,
-	groceries: { track: true },
-	bills,
-	other
-  }
-}
+create sheet monthly_budget
+  add date September 2019,
```

Comment:
doesn't this violate the single lookahead thing? And also what was last on the grammar in slack? The "create" token was for a spreadsheet, then "add" was for a sheet only

Reply:
I also don't see this being wrapped in anything that signifies it creates a spreadsheet document. i.e. including "create SHEET_NAME sheet+"